compose/all/compose.yml

```diff
@@ -130,6 +130,7 @@ services:
       DATAFED_REPO_ID_AND_DIR: "${DATAFED_REPO_ID_AND_DIR}"
       DATAFED_GLOBUS_SUBSCRIPTION: "${DATAFED_GLOBUS_SUBSCRIPTION}"
       DATAFED_GLOBUS_CONTROL_PORT: "${DATAFED_GLOBUS_CONTROL_PORT}"
+      DATAFED_GCS_COLLECTION_BASE_PATH: "/"
```

Comment:
This needs to also be added to compose/repo/compose.yml, also you might want to import it from .env file instead of hardcoding it here. That will mean you need to make changes in the generate_env scripts.